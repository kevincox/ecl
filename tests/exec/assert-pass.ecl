assert 1 == 1
dict = {
	foo = "value"
	assert foo == bar
	bar = "value"
}
