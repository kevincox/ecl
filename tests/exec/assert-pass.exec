cmd: load assert-pass.ecl -flines
status: 0
