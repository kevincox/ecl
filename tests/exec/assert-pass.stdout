	DICT	{
dict	DICT	{
dict.bar	STR	"value"
dict.foo	STR	"value"
dict	DICT	}
	DICT	}
